### ISSUE TITLE
"Your app is using the "force-ssl" package, but ssl is not setup in your mup config."

### ISSUE THREAD

**Issue by nadeemja** [zodern/meteor-up#553]
Hi,

Using latest version as of now. 

I have the ssl option set correctly (worked previously without change).

Still I'm getting: Your app is using the "force-ssl" package, but ssl is not setup in your mup config.

And a 502 Bad Gateway even though everything says "Successful" during deploy.

**Comment by zodern** [comment]
Do you remember what version were you using before? Could you please share the `meteor` section of your config?

**Comment by skynightz93** [comment]
Same problem here

**Comment by zodern** [comment]
@nizulzaim could you please share the meteor section of your config? Is ssl also not working for you?

**Comment by andy4545** [comment]
I am getting the same issue - suddenly after installing latest zodern/meteor-up, with latest meteor today.

```
module.exports = {
   meteor: {
       ssl: {
        crt: '/home/ubuntu/projname/.deploy/mycert.crt', 
        key: '/home/ubuntu/projname/.deploy/mykey.key', 
        port: 443 
       },
   }
}
```

**Comment by andy4545** [comment]
```
meteor: {
    name: 'projname',
    path: '../app',
    servers: {
      one: {}
    },
    buildOptions: {
      debug: true,
      serverOnly: true,
    },
    env: {
        ROOT_URL: 'https://projecturl.org',
        MONGO_URL: 'mongodb://ipaddressofmongoserver/meteordb'
    },
    docker: {
        image: 'abernix/meteord:base',
    },
   ssl: {
        crt: '/home/ubuntu/projname/.deploy/mycert.crt', 
        key: '/home/ubuntu/projname/.deploy/mykey.key', 
        port: 443 
       },
   deployCheckWaitTime: 60
  },
```

**Comment by andy4545** [comment]
IP] - Verifying Deployment
[IP] x Verifying Deployment: FAILED
        
        -----------------------------------STDERR-----------------------------------
        bin/node" "/usr/bin/npm" "install" "--unsafe-perm"
        npm ERR! node v4.8.0
        npm ERR! npm  v3.10.10
        npm ERR! code ELIFECYCLE
        npm ERR! meteor-dev-bundle@0.0.0 install: `node npm-rebuild.js`
        npm ERR! Exit status 1
        npm ERR! 
        npm ERR! Failed at the meteor-dev-bundle@0.0.0 install script 'node npm-rebuild.js'.
        npm ERR! Make sure you have the latest version of node.js and npm installed.
        npm ERR! If you do, this is most likely a problem with the meteor-dev-bundle package,
        npm ERR! not with npm itself.
        npm ERR! Tell the author that this fails on your system:
        npm ERR!     node npm-rebuild.js
        npm ERR! You can get information on how to open an issue for this project with:
        npm ERR!     npm bugs meteor-dev-bundle
        npm ERR! Or if that isn't available, you can get their info via:
        npm ERR!     npm owner ls meteor-dev-bundle
        npm ERR! There is likely additional logging output above.
        
        npm ERR! Please include the following file with any support request:
        npm ERR!     /bundle/bundle/programs/server/npm-debug.log
        -----------------------------------STDOUT-----------------------------------
        
        To see more logs type 'mup logs --tail=50'
        
        ----------------------------------------------------------------------------

**Comment by andy4545** [comment]
I solved this issue by uninstalling bcrypt 

`meteor npm uninstall bcrypt --save
`

It should be ok to ignore the warning in your development that asks you to install bcrypt, otherwise  you can save-dev to run it in development mode only. 

> 
> Note: you are using a pure-JavaScript implementation of bcrypt.
> While this implementation will work correctly, it is known to be
> approximately three times slower than the native implementation.
> In order to use the native implementation instead, run
> 
>  meteor npm install --save bcrypt

Read this issue for ref: #369

**Comment by CrackedBeefcake** [comment]
+1 for the issue

**Comment by mackbrowne** [comment]
I'm having this issue as well.  Uninstalling bcrypt didn't help.

I updated from mup 1.2.4 -> 1.2.5 and started getting this error.

UPDATE:
I reverted back to mup 1.2.4 to fix my problem.  I wouldn't recommend upgrading mup at this time.

Using digitalocean's Ubuntu 14.04.5 image and the abernix/meteord:base docker image

**Comment by CrackedBeefcake** [comment]
i just ignored it and everything works fine. the alert is irritating tho

**Comment by tomtom87** [comment]
+1 having this issue cannot deploy new app to production now... will try mup 1.2.4 see if this fixes it

**Comment by nadeemja** [comment]
Hi again,

Upgrading from 512MB to a 1024MB VPS fixed it (I saw a ran out of memory error somewhere).

Using 1.2.5 successfully now.

**Comment by tomtom87** [comment]
@nadeemja ahhh ok I will try this.. 1.2.4 is working fine by the way

**Comment by zodern** [comment]
I just published 1.2.6 which fixed the warning appearing when it shouldn't.
